hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyPurgeRequest.java

```diff
@@ -115,18 +126,42 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
               bucketName);
         }
       }
+
+      if (result == Result.REPLAY) {
+        LOG.debug("Replayed Transaction {}. Request: {}", trxnLogIndex,
+            purgeKeysRequest);
+        if (!keysNotPurged.isEmpty()) {
```

Comment:
NIT: We can use the Java8 String method String.join(", list)

Reply:
And also as discussed offline for debug statements we can use isDebugEnabled, so that parameter conversion will not happen.